tools/api-markdown-documenter/README.md

```diff
@@ -144,28 +144,54 @@ As noted above, this library is intended to be consumed programmatically.
 While we may at some point add some command line interfaces for common paths, the primary goal of this library's architecture is to be flexible and extensible.
 To that end, we have broken its logic into a few isolable steps, each offering its own extensibility offerings.
 
-End to end, this library can be viewed as a psuedo-functional transformation pipeline mapping from an [APIModel][] generated by `API-Extractor` to one or more `Markdown`-formatted document files.
+End to end, this library can be viewed as a psuedo-functional transformation pipeline mapping from an [APIModel][] generated by `API-Extractor` to one or more `Markdown` or `HTML` formatted document files.
```

Comment:
```suggestion
End to end, this library can be viewed as a pseudo-functional transformation pipeline mapping from an [APIModel][] generated by `API-Extractor` to one or more `Markdown` or `HTML` formatted document files.
```

Also, curious, does the empty "link target" for APIModel serve a purpose?

Reply:
That's a supported syntax in GitHub flavored markdown. It corresponds to a definition at the bottom of the file. Lets you reuse link targets without having to specify them everywhere.

Reply:
Oh, I know those, but didn't know you could make the text itself the "id". I see now 👍.